app/models/SearchManagementRepository.scala

```diff
@@ -56,6 +56,26 @@ class SearchManagementRepository @Inject()(dbapi: DBApi, toggleService: FeatureT
     InputTag.insert(inputTag)
   }
 
+  def listAllUsers(): Seq[User] = db.withConnection { implicit connection =>
```

Comment:
I wonder about adding users and teams etc to SearchManagementRepository?  Is there another pattern we could use?   Seems likewe add EVERYTHING to APIController, then add EVERYTHING to SearchManagementRepository.

Maybe a UserManagementRepository?  Dunno, maybe worth thinking about.

Reply:
Good point, I leave that decision with the technical architect...